service/src/intTest/java/uk/nhs/adaptors/gpc/consumer/gpc/GpcConfigurationValidationTest.java

```diff
@@ -0,0 +1,259 @@
+package uk.nhs.adaptors.gpc.consumer.gpc;
+
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.context.annotation.Configuration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@SpringBootTest
+public class GpcConfigurationValidationTest {
```

Comment:
Just an observation, in general as the integration tests are usually quite slow some of the tests can be done as small unit tests.

Reply:
To test that the application fails to start when the properties are invalid, it needs to try to start the application and allow the properties to be configured, using a test application context runner, there may be a need for separate unit tests as well for each validator now that we are breaking the single one down, but we will always need to test the above